quiche/src/recovery/gcongestion/recovery.rs

```diff
@@ -856,10 +856,12 @@ impl RecoveryOps for GRecovery {
         r.rtt() + (r.rttvar() * 4).max(GRANULARITY)
     }
 
+    /// The most recent data delivery rate estimate in bytes/s.
```

Comment:
Any tests that we can run against both implementations in order to validate that they mostly compute the same thing?

Reply:
I added a test for congestion https://github.com/cloudflare/quiche/pull/2043/commits/68b1bc6c493dfc7b30bf23a393092563cdb18e85

gcongestion operates very differently and I wasn't able to easily add a test that increases the delivery_rate. I am in the process of reverse engineering how gcongestion operates so I can control it.